src/Animations/babylon.animation.ts

```diff
@@ -1,8 +1,21 @@
 ﻿module BABYLON {
+    /**
+     * Represents the range of an animation.
+     */
     export class AnimationRange {
+        /**
+         * Initializes the range of an animation.
+         * @param {string} name - The name of the animation range.

```

Comment:
Types must not be set (they are inferred)

Reply:
Fixed